about.html

```diff
@@ -0,0 +1,81 @@
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#">Navbar</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarTogglerDemo02">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="/index.html">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/about.html">About me</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/contact.html">Contact me</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/projects.html">projects</a>
+                        </li>
+                    </ul>
+                    <!-- <form class="d-flex">
+                  <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                  <button class="btn btn-outline-success" type="submit">Search</button>
+                </form> -->
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <h1>About me</h1>
+        <img src="assets/moi.jpeg" alt="picture of me">    
+        <p>Aspiring 3D artist turned web designer. I would describe myself as a pragmatic and efficient man. I enjoy playing various games, including card games, video games, board games, etc. Feel free to write me if you are in need of a comission!</p>
+        <a class="btn btn-dark" href="linkedIn" role="button">Resume on Linkedin</a>
+        <a class="btn btn-outline-secondary" href="contact.html" role="button">Contact me</a>
+    </main>
+    <footer>
+        <div class="p-3 mb-2 bg-dark text-white">
+            <ul class="nav justify-content-center">
+                <li class="nav-item">
+                    <nav class="nav flex-column" style="margin-right: 10px;">
+                        <a href="about.html" class="text-white bg-dark">About</a>
+                        <a href="contact.html" class="text-white bg-dark">Contact</a>
+                        <a href="projects.html" class="text-white bg-dark">Projects</a>
+                    </nav>
+                </li>
+                <li class="nav-item">
+                    <a href="https://www.artstation.com/walldo">
+                        <img src="assets/Artstation.png" style="max-width: 70px; margin: 0px 10px;" alt="artstation" />
```

Comment:
Were your images correctly exported? Also, try to use classes instead inline styles.

![image](https://user-images.githubusercontent.com/6362750/235269078-261679a2-e82c-4785-acd9-cefe8a3fd505.png)